File: service/src/main/java/coding/challenge/core/repository/UserPhraseRepository.java
package coding.challenge.core.repository;

import coding.challenge.core.model.UserPhrase;
import org.springframework.data.repository.CrudRepository;

/**
 * Represents repository for management of {@link UserPhrase} entity.
 */
public interface UserPhraseRepository extends CrudRepository<UserPhrase, UserPhrase.UserPhraseId> {
}

File: service-kt/src/main/kotlin/coding/challenge/core/repository/UserPhraseRepository.kt
package coding.challenge.core.repository

import coding.challenge.core.model.UserPhrase
import org.springframework.data.repository.CrudRepository

/**
 * Represents repository for management of [UserPhrase] entity.
 */
interface UserPhraseRepository : CrudRepository<UserPhrase, UserPhrase.UserPhraseId>
